addons/voxel-core/engine/importers/meshes.gd

```diff
@@ -5,7 +5,12 @@ extends EditorImportPlugin
 
 
 ## Enums
-enum Presets { DEFAULT }
+enum Presets {
```

Comment:
I see that these presets are repeated in multiple EditorImportPlugins, maybe it would be best to make a parent class for similar EditorImportPlugins, I feel it's best to keep the code DRY.